### PR TITLE
correcting simplenificluster.yaml

### PR DESCRIPTION
**Pull request by PDEUXA**
| Q               | A
| --------------- | ---
| Bug fix?        |yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets | https://github.com/konpyutaika/nifikop/issues/94
| License         | Apache 2.0

### What's in this PR?
simplenificluster.yaml is not a working example as it is currently. (https://github.com/konpyutaika/nifikop/issues/94)
Not really sure was is the problem with the current conf iguration(I suspect the nifi version (1.15 is https by defaults, and simplenificluster points to http)).

### Why?

Issue was causing nifi node to indefinitely restart, due to nodes cannot reaches the headless service. Chart is a little bit complex, but maybe there is some unnecessary content.

### Checklist

- [X] Implementation tested
- [X] Error handling code meets the [guideline](docs/error-handling-guide.md)
- [X] Logging code meets the guideline
- [X] User guide and development docs updated (if needed)
- [] Append changelog with changes

### CODE REVIEW

File: config/samples/simplenificluster.yaml [hunk 1]
```diff
@@ -5,13 +5,31 @@ metadata:
...
     headlessEnabled: true
-  zkAddress: "zookeeper.zookeeeper:2181"
-  zkPath: "/simplenifi"
-  clusterImage: "apache/nifi:1.15.3"
-  oneNifiNodePerNode: false
+    labels:
+      cluster-name: simplenifi
+  zkAddress: "zookeeper.zookeeper:2181"
+  zkPath: /simplenifi
+  externalServices:
```
Comment (michael81877) [inline config/samples/simplenificluster.yaml:12]: IMO, should probably retain all of the example `externalServices` as sample of what they each look like.

File: config/samples/simplenificluster.yaml [hunk 2]
```diff
@@ -38,49 +83,23 @@ spec:
   propagateLabels: true
   nifiClusterTaskSpec:
     retryDurationMinutes: 10
+  readOnlyConfig:
+    zookeeperProperties:
+      overrideSecretConfig:
+        data: zookeeper.properties
+        name: simplenifi
```
Comment (michael81877) [inline config/samples/simplenificluster.yaml:90]: If the intent is for this to work out of the box on a clean cluster, then i believe this `overrideSecretConfig` will cause it to fail. the `simplenifi` secret would need to exist in the nifi namespace for this to work.

File: config/samples/simplenificluster.yaml (hunk 1 above)
Comment (PDEUXA) [reply config/samples/simplenificluster.yaml:12]: Kind succeed to make the cluster work with all example of externalServices. What is the purpose of exposing this, every K8S user should  already be aware of service type?

File: config/samples/simplenificluster.yaml (hunk 2 above)
Comment (PDEUXA) [reply config/samples/simplenificluster.yaml]: I do not fail, but I removed it as the cluster can run without this.

File: config/samples/simplenificluster.yaml (hunk 1 above)
Comment (michael81877) [reply config/samples/simplenificluster.yaml:12]: I do agree with that. I don't think it's strictly necessary, but it just serves as an example of what each service type looks like in a `NifiCluster` spec.

Comment (erdrix) [reply config/samples/simplenificluster.yaml:12]: For the simplenificluster we can keep it as short as possible, the details should be in the https://github.com/konpyutaika/nifikop/blob/master/config/samples/nifi_v1alpha1_nificluster.yaml file

File: config/samples/simplenificluster.yaml [hunk 3]
```diff
@@ -5,13 +5,31 @@ metadata:
...
+    - metadata:
+        labels:
+          cluster-name: driver-simplenifi
+      name: driver-ip
+      spec:
+        portConfigs:
+          - internalListenerName: http
+            port: 8080
+        type: LoadBalancer
+  clusterImage: 'apache/nifi:1.12.1'
```
Comment (erdrix) [inline config/samples/simplenificluster.yaml:22]:
```suggestion
  clusterImage: "apache/nifi:1.15.3"
```
The latest version tested

**Review by erdrix — CHANGES_REQUESTED** [review]

**Review by erdrix — APPROVED** [review]

**Pull request merged**